prestodb/client.py

```diff
@@ -222,6 +231,13 @@ def http_headers(self):
             '{}={}'.format(name, value)
             for name, value in self._client_session.properties.items()
         )
+
+        # merge custom http headers
+        for key in self._client_session.headers:
+            if key.startswith(constants.HEADER_PREFIX):
+                raise ValueError('invalid HTTP header {}'.format(key))
```

Comment:
It would help to know why the header is invalid:

```python
raise ValueError('invalid HTTP header {}: must not start with "{}"'.format(
    key,
    constants.HEADER_PREFIX,
))